purchases/src/main/kotlin/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -326,13 +339,19 @@ internal class Backend(
         )
         val call = object : Dispatcher.AsyncCall() {
             override fun call(): HTTPResult {
+                val body = mapOf(
+                    APP_USER_ID to appUserID,
+                    NEW_APP_USER_ID to newAppUserID,
+                )
+                val postFieldsToSign = listOf(
+                    APP_USER_ID to appUserID,
+                    NEW_APP_USER_ID to newAppUserID,
+                )
```

Comment:
I thought about just copying the `body` (modifying it to be a list of key-values) but in the end, I thought it's better to keep it as a separate entity.